StripeCore/StripeCore/Source/DownloadManager/DownloadManager.swift

```diff
@@ -9,37 +9,25 @@ import UIKit
 
 /// For internal SDK use only.
 @objc(STP_Internal_DownloadManager)
-@_spi(STP) public class DownloadManager: NSObject, URLSessionDelegate {
+@_spi(STP) public class DownloadManager: NSObject {
     public typealias UpdateImageHandler = (UIImage) -> Void
 
-    private enum Error: Swift.Error {
-        case downloadSyncFailure
-        case downloadAsyncFailure
+    enum Error: Swift.Error {
+        case failedToMakeImageFromData
     }
 
     public static let sharedManager = DownloadManager()
 
-    let downloadQueue: DispatchQueue
-    let downloadOperationQueue: OperationQueue
-    let session: URLSession!
+    let imageCache = ImageCache()
 
-    var imageCache: [String: UIImage]
-    var pendingRequests: [String: URLSessionTask]
-    var updateHandlers: [String: [UpdateImageHandler]]
-
-    let imageCacheSemaphore: DispatchSemaphore
-    let pendingRequestsSemaphore: DispatchSemaphore
-
-    let STPCacheExpirationInterval = (60 * 60 * 24 * 7)  // 1 week
-    var urlCache: URLCache?
+    private let session: URLSession
+    private let analyticsClient: STPAnalyticsClient
+    private var diskCache: URLCache?
```

Comment:
nit: This can be a let

Reply:
Based on this [comment](https://github.com/stripe/stripe-ios/pull/3487#discussion_r1559955101), this ended up being removed!